themes-default/slim/src/components/index.js

```diff
@@ -5,6 +5,7 @@ export { default as AnidbReleaseGroupUi } from './anidb-release-group-ui.vue';
 export { default as AppHeader } from './app-header.vue';
 export { default as Backstretch } from './backstretch.vue';
 export { default as Config } from './config.vue';
+export { default as ConfigBackuprestore } from './config-backuprestore.vue';
```

Comment:
This makes more sense in my opinion... :thinking: 
```suggestion
export { default as ConfigBackupRestore } from './config-backup-restore.vue';
```